### DISCUSSION TITLE
Support for Firebase v9

### DISCUSSION THREAD

**Discussion by paul-uae** [prescottprue/react-redux-firebase#1220] [Q&A]
Hi All,

I am evaluating if this library is the right fit for my project and was wondering if there are any plans to move to the new Firebase v9 modular API? 

Regards
Paul

**Comment by prescottprue** [comment]
Hey @paul-uae - I don't as actively use this library myself anymore since my main projects aren't using Firebase + Redux as much as they were previously. [redux-firestore](https://github.com/prescottprue/redux-firestore/pulls) would probably get the update first since it is smaller and Firestore is often a more common choice than RTDB these days

That said some other libs I made [like cypress-firebase](https://github.com/prescottprue/cypress-firebase) aren't based around redux, so those will be updated first.

Always open to PRs even if it is a major breaking change if anyone wants to take this on.

**Comment by paul-uae** [comment]
Hi Scott,

Thanks so much for taking the time to respond. I really appreciate the effort.

I found some other ways to achieve my goals in this project so am good for now.

Take care.


Best Regards

Paul Trotter
https://nz.linkedin.com/in/paulstrotter
UAE Cell: +971 900 6924
NZ Cell: +64 21 379 389
________________________________
